src/lib/components/backoffice/Sidebar/Sidebar.js

```diff
@@ -53,6 +54,20 @@ class Sidebar extends Component {
           <Overridable id="Sidebar.Menu">
             <>
               <Menu text vertical className="bo-menu">
+                <Menu.Item>
+                  <Menu.Header>Actions</Menu.Header>
+                  <Menu.Menu>
+                    <Menu.Item
+                      as={Link}
+                      active={checkinActive}
+                      to={BackOfficeRoutes.checkIn}
+                    >
+                      Check-in
+                      <div className="smaller-font">Return copies</div>
```

Comment:
in terms of class name please change it to something like `menu-item-description`. The `smaller-font` is a bit misleading, because it does not tell us where it is used or we don't know smaller than which font specifically.